tell_me_your_secrets/__main__.py

```diff
@@ -42,30 +47,17 @@
 argument_parser.add_argument('-g', '--gitignore', help='Ignore .gitignore mapped objects. ', action='store_true')
 module_logger = create_logger(MODULE_NAME)
 
-# Options
```

Comment:
I can leave this back but I Felt like it was redundant with the argument config

Reply:
We can remove options.  I wanted to keep the process comments to ensure faster readability of code.

Reply:
Sure will update